### PR TITLE
[10.0][MIG][base_export_manager] Full migration to version 10

### PR DESCRIPTION
**Pull request by yajo**
I was becoming crazy with git... versions 8 and 9 of the addon diverged.

Now, features from both versions will be available for 10.0.

I'm thinking on removing transbot commits, since translations are gonna be broken anyways (either missing v9 strings or v8 ones).

This resumes work form https://github.com/OCA/server-tools/pull/712.

@Tecnativa

### CODE REVIEW

File: base_export_manager/tests/test_ir_exports_line.py [hunk 1]
```diff
@@ -29,8 +37,52 @@ def test_get_label_string(self):
         m_ir_exports_line = self.env['ir.exports.line']
         export_line = m_ir_exports_line.create({'name': 'parent_id/name',
                                                 'export_id': self.export.id})
+        check_label = " (res.partner) (parent_id/name)"
         self.assertEqual(export_line.with_context(lang="en_US").label,
-                         "Related Company/Name (parent_id/name)")
+                         "Related Company (res.partner)/Name" + check_label)
```
Comment (yvaucher) [inline base_export_manager/tests/test_ir_exports_line.py]: This seems to fail:

```
FAIL: test_get_label_string (odoo.addons.base_export_manager.tests.test_ir_exports_line.TestIrExportsLineCase)
Traceback (most recent call last):
`   File "/home/travis/build/OCA/server-tools/base_export_manager/tests/test_ir_exports_line.py", line 42, in test_get_label_string
`     "Related Company (res.partner)/Name" + check_label)
` AssertionError: u'Related Company/Name (parent_id/name)' != 'Related Company (res.partner)/Name (res.partner) (parent_id/name)'
FAILED
Module base_export_manager: 1 failures, 0 errors
```

**Review by yvaucher — CHANGES_REQUESTED** [review]
Just the test to fix. Code review :+1: + tested locally (only the export access right and the visibility of the button didn't try export profiles)

File: base_export_manager/models/ir_exports_line.py [hunk 2]
```diff
@@ -3,8 +3,8 @@
 # Copyright 2015-2016 Jairo Llopis <jairo.llopis@tecnativa.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from openerp import models, fields, api, exceptions
-from openerp.tools.translate import _
+from odoo import models, fields, api, exceptions
+from odoo.tools.translate import _
```
Comment (pedrobaeza) [inline base_export_manager/models/ir_exports_line.py]: Import `_` also from `odoo`

File: base_export_manager/__manifest__.py [hunk 3]
```diff
@@ -25,8 +24,8 @@
               'LasLabs, '
```
Comment (pedrobaeza) [inline base_export_manager/__manifest__.py]: Remove Antiun of this list

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Little changes, but overall OK

**Review by chienandalu — APPROVED** [review]
Tested funcionality:
- Create an export list and check that it's added to the new export view.
- Create an export from the new view and check that appears on its model view.
- Restrict export for a users group and check that export is not available anymore on the affected views.

File: base_export_manager/__manifest__.py (hunk 3 above)
Comment (Yajo) [inline base_export_manager/__manifest__.py]: done

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@yvaucher please give your approval.

**Comment by pedrobaeza** [comment]
As @yvaucher's comment is attended and he doesn't answer, I proceed to merge.

**Pull request merged**